doctorapp/src/components/home/Appointment.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <div class="appointment">
+      <div class="appointment-header">
+        <span class="appointment-author">Appointment</span>
+      </div>
+      <div class="appointment-content">
+        <div>
+          <p><strong>Time:</strong> {{ time }}</p>
+          <p><strong>Date:</strong> {{ date }}</p>
+        </div>
+        <div>
+          <p><strong>Type:</strong> {{ type }}</p>
+          <p><strong>Location:</strong> {{ location }}</p>
```

Comment:
Nitpick: A `b` tag is more suitable here (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong)